### PR TITLE
PAYEE-585: Updated error template no auth + spec

### PR DESCRIPTION
**Pull request by PerrettJ4**
- [ ] Unit tests passing
- [ ] Coverage reached
- [ ] Acceptance tests passing
- [ ] Reviewed

### CODE REVIEW

File: app/views/ErrorTemplateNoauth.scala.html [hunk 1]
```diff
@@ -16,18 +16,18 @@
...
 
-@this(mainTemplate: MainTemplate)
+@this(mainTemplate: MainTempleteNew)
 
 @(pageTitle: String, heading: String, message1: String, additionalMessages: List[String])(implicit request: Request[_],
         messages: Messages, templateRenderer: uk.gov.hmrc.renderer.TemplateRenderer, ec: ExecutionContext)
 
 @mainTemplate(
     title=pageTitle,
-    hidePtaAccountNav = true
```
Comment (masahmad) [inline app/views/ErrorTemplateNoauth.scala.html:26]: you prob need to add  showPtaAccountNav = false as  this is in the new main template - please check my branch payee-587

Comment (PerrettJ4) [reply app/views/ErrorTemplateNoauth.scala.html:26]: sure thing, will update now!

File: test/controllers/ErrorPagesHandlerSpec.scala [hunk 2]
```diff
@@ -65,7 +65,7 @@ class ErrorPagesHandlerSpec extends BaseSpec {
       val title = doc.select("title").text()
       title must include("Bad request - 400")
 
-      val heading = doc.select(".h1-heading").text()
+      val heading = doc.select(".govuk-heading-l").text()
```
Comment (sergi-hmrc) [inline test/controllers/ErrorPagesHandlerSpec.scala:68]: the original idea was to have checkSomethingNew + checkSomething together so no test would be broken. Once everything was migrated big refactor to remove New from all methods

File: app/views/ErrorTemplateNoauth.scala.html [hunk 3]
```diff
@@ -16,18 +16,19 @@
...
+    backLinkContent = None,
+    showPtaAccountNav = false
 ){
-    <div class="section">
-        <h1 class="h1-heading">@heading</h1>
-        <div class="grid grid-2-3">
+    <div class="govuk-grid-row">
+        <h1 class="govuk-heading-l">@heading</h1>
+        <div class="govuk-grid-column-two-thirds">
```
Comment (brendanstraughan) [inline app/views/ErrorTemplateNoauth.scala.html:31]: 31 and 30 need to be swapped around, the formatting doesnt look quite right. So like
`<div class="govuk-grid-row">
        <div class="govuk-grid-column-two-thirds">
            <h1 class="govuk-heading-l">@heading</h1>`

File: app/views/ErrorTemplateNoauth.scala.html [hunk 4]
```diff
@@ -16,18 +16,19 @@
...
             <p>@Html(message1)</p>
```
Comment (brendanstraughan) [inline app/views/ErrorTemplateNoauth.scala.html:32]: p tags need a 
`class="govuk-body"`

File: app/views/ErrorTemplateNoauth.scala.html (hunk 3 above)
Comment (PerrettJ4) [reply app/views/ErrorTemplateNoauth.scala.html:31]: Ahh okay, good spot, I will amend when home

**Review by masahmad — APPROVED** [review]

**Review by sergi-hmrc — APPROVED** [review]

**Review by brendanstraughan — APPROVED** [review]

**Pull request merged**